mlir/lib/Dialect/Affine/Analysis/AffineStructures.cpp

```diff
@@ -222,8 +221,17 @@ LogicalResult FlatAffineValueConstraints::addBound(BoundType type, unsigned pos,
   fullyComposeAffineMapAndOperands(&map, &operands);
   map = simplifyAffineMap(map);
   canonicalizeMapAndOperands(&map, &operands);
-  for (auto operand : operands)
+  for (auto operand : operands) {
+    if (!((isTopLevelValue(operand)) || (isAffineInductionVar(operand)))) {
+      emitError(
+          operand.getLoc(),
+          " Above Val is neither defined as a Top-Level Value nor is an "
+          "AffineInductionVar. Non-terminal symbol / loop IV expected. Pass "
+          "Aborted!");
```

Comment:
Remove unnecessary capitatlization. All emit messages should start in lower case.

"value is neither a top-level value nor an affine IV; non-termnial symbol/loop IV expected; pass aborted"